docs/content/mobile/ios/cloud-deploy.md

```diff
@@ -48,6 +64,10 @@ Note: You don’t need to enter anything in the text field.
 
 ## iOS device{#ios-device}
 
+To connect your mobile application to a cloud deployment, you need to authenticate against our services.
+This guide describes how to authenticate using the development authentication flow which we provide for early stages in development.
+Alternatively, if you want to create your own authentication server, follow [this guide](https://docs.improbable.io/reference/latest/shared/auth/integrate-authentication-platform-sdk).
```

Comment:
note that we do not provide an out of the box integration for this right now. If people implement this, they need to do it on their own.

not sure if we should highlight this?

Reply:
I think this is fine for now, our level of support will be clarified further when we publish further (planned) docs about this.